### PR TITLE
proposed changes

### PR DESCRIPTION
**Pull request by sternhenri**
Referenced from https://github.com/sourcecred/mission/issues/4#issuecomment-448676284

Feel free to ignore, just in case it helps ;)

### CODE REVIEW

**Comment by svlandeg** [comment]
@sternhenri : I commented on two typos as well in the already merged PR https://github.com/sourcecred/mission/pull/6, maybe you can include those corrections here as well?

**Comment by sternhenri** [comment]
done! thank you :)

File: overview.md [hunk 1]
```diff
@@ -11,13 +11,15 @@ However, as this idea evolved, we ran into a big problem: we don't know who
 deserves credit in an open-source project, and thus we don't know how to
 distribute the rewards. The obvious approaches, like counting commits or lines
 of code, would be far too easy to game—and would in any case miss out on vital
-work like design or issue triaging.
+work like design or issue triaging, not to mention that it would miss out on the
+underlying quality of the commits or code.
```
Comment (teamdandelion) [inline overview.md]: Even before the modification, this sentence is pretty ungainly:
The obvious approaches (descriptive subject)
like counting commits or lines of code (examples provided)
would be easy to game (passive tense property of the subject)
-and would in any case miss out on vital work (another property of the subject)
like design or issue triaging (examles of another property)

so I think adding in another clause with more examples makes it kind of unmanageable. I think pointing out that the naive approaches are deficient w.r.t picking up the right kinds of work *and* discerning the quality of work is a valuable one, but I don't want to overload this sentence. could you re-write to factor these concepts apart more clearly?

File: overview.md [hunk 2]
```diff
@@ -11,13 +11,15 @@ However, as this idea evolved, we ran into a big problem: we don't know who
...
 
 This inability to quantify contributions accurately affects every method for
 funding open-source. Projects need a fair system for deciding how to distribute
 rewards regardless of whether they come through sponsorships, donations,
 [cryptocurrency airdrops], or through any [other means]. SourceCred is designed
-to do just that.
```
Comment (teamdandelion) [inline overview.md]: I like the punchiness of "SourceCred is designed to do just that". Happy to have the additional concept in a separate sentence, e.g.:
"SourceCred is designed to do just that. We hope that these quantitative measurements of value will enable downstream work to fund open-source contributors".
(although I think this proposed sentence is also a little vague)

File: overview.md [hunk 3]
```diff
@@ -60,14 +62,14 @@ that just say "me too".
...
-SourceCred is currently in a early beta stage, with much of the core
+SourceCred is currently in an early beta stage, with much of the core
 infrastructure in place, and a functioning prototype. Anecdotally, SourceCred
 seems to do a good job of assigning cred to the major contributors of GitHub
 projects; we've shown contributors' cred totals to the maintainers of several
 projects, and the feedback has been quite positive. However, SourceCred doesn't
 yet do a great job of assigning cred to individual pull requests, issues, or
 commits. There isn't yet much data distinguishing pulls or commits from each
-other, so whichever ones get more discussion often to have the highest scores.
+other, so whichever ones get more discussion often get to have the highest scores.
```
Comment (teamdandelion) [inline overview.md]: s/get to have/get/

File: overview.md [hunk 4]
```diff
@@ -142,7 +144,8 @@ people are gaming.
 Second, since SourceCred is itself open-source, anyone can propose new heuristics to
 mitigate gaming. For example, the community could develop a machine learning model
 to distinguish between threads that are constructive and threads that are flame wars,
-and then greatly reduce the weight on the flame wars.
```
Comment (teamdandelion) [inline overview.md:145]: Again, I think the added concept would be better expressed as a followon sentence. That way this sentence makes one point clearly, and the next sentence follows with another point clearly

File: overview.md [hunk 5]
```diff
@@ -158,4 +161,7 @@ SourceCred is being developed with funding and support from [Protocol Labs].
 
 You can think of SourceCred as the "IPFS" to SourceGrain's "Filecoin".
 SourceCred is building a protocol for credit attribution independent of
-economic incentives, and SourceGrain will add an incentive layer on top.
+economic incentives, and SourceGrain will add an incentive layer on top
+using SourceCred. They can nonetheless exist independently from one
+another, for instance with SourceCred being used for other purposes,
+or SourceGrain being used atop another credit attribution system.
```
Comment (teamdandelion) [inline overview.md:166]: LGTM

**Review by teamdandelion — COMMENTED** [review]
Looks good overall, some changes in sentence structure proposed

**Comment by sternhenri** [comment]
Another pass :) Feel free to merge or amend!

**Comment by teamdandelion** [comment]
Looks good. Thank you @sternhenri :)

**Pull request merged**